### PR TITLE
Consolidated PR for #109 to #112, #139, #135. #137, #138

### PR DESCRIPTION
**Pull request by VisLab**

### CODE REVIEW

**Comment by VisLab** [comment]
This is a consolidated PR for the items that the HWG has discussed for updates to 8.3.0.  Please review carefully for discussion.

**Comment by IanCa** [comment]
I don't see any mistakes in the units/values/etc sections from what we discussed.

Are we still planning to update the allowedCharacters on value classes for this release?

**Comment by VisLab** [comment]
> I don't see any mistakes in the units/values/etc sections from what we discussed.
> 
> Are we still planning to update the allowedCharacters on value classes for this release?

Yes -- haven't gotten to it yet.

Attached is a summary of the changes --- as generated by the new schema comparison tool (Thx @IanCa).  In the current PR, I just changed `degree Celsius` to `degree-Celsius` as the unit wasn't supposed to have a space according to the rules.  However, I am wondering whether I should leave both in and add the `deprecatedFrom=8.3.0` to `degree Celsius`.
[HED8.2.0_HED8.3.0_differences.txt](https://github.com/hed-standard/hed-schemas/files/14373705/HED8.2.0_HED8.3.0_differences.txt)

**Review by neuromechanist — APPROVED** [review]
LGTM 😊

File: standard_schema/prerelease/HED8.3.0.mediawiki [hunk 1]
```diff
@@ -487,13 +487,16 @@ Each term in this vocabulary has a human-readable description and may include ad
 **** Female <nowiki>[Biological sex of an individual with female sexual organs such ova.]</nowiki>
 **** Male <nowiki>[Biological sex of an individual with male sexual organs producing sperm.]</nowiki>
 **** Intersex <nowiki>[Having genitalia and/or secondary sexual characteristics of indeterminate sex.]</nowiki>
+**** Other-sex <nowiki>[A non-specific designation of sexual traits.]</nowiki>
 *** Ethnicity <nowiki>[Belong to a social group that has a common national or cultural tradition. Use with Label to avoid extension.]</nowiki>
```
Comment (smakeig) [inline standard_schema/prerelease/HED8.3.0.mediawiki:491]: Shouldnt this refer to a population or culture within which the ethnicity references?

Comment (VisLab) [reply standard_schema/prerelease/HED8.3.0.mediawiki:491]: Could you make a specific wording suggestion.  Ethnicity isn't in BIDS so we are not constrained by a particular definitions.   Thx @smakeig

**Review by happy5214 — CHANGES_REQUESTED** [review]

File: standard_schema/prerelease/HED8.3.0.mediawiki [hunk 2]
```diff
@@ -673,15 +673,17 @@ Each term in this vocabulary has a human-readable description and may include ad
...
-******* <nowiki># {takesValue, valueClass=numericClass, unitClass=physicalLengthUnits}</nowiki> 
-****** Length <nowiki>[The linear extent in space from one end of something to the other end, or the extent of something from beginning to end.]</nowiki> 
-******* <nowiki># {takesValue, valueClass=numericClass, unitClass=physicalLengthUnits}</nowiki> 
-****** Width <nowiki>[The extent or measurement of something from side to side.]</nowiki>
 ******* <nowiki># {takesValue, valueClass=numericClass, unitClass=physicalLengthUnits}</nowiki>
 ****** Height <nowiki>[The vertical measurement or distance from the base to the top of an object.]</nowiki>
+******* <nowiki># {takesValue, valueClass=numericClass, unitClass=physicalLengthUnits}</nowiki>
+****** Length <nowiki>[The linear extent in space from one end of something to the other end, or the extent of something from beginning to end.]</nowiki> 
 ******* <nowiki># {takesValue, valueClass=numericClass, unitClass=physicalLengthUnits}</nowiki> 
+****** Radius <nowiki>[The distance of the line from the center of a circle or a sphere to its perimeter or outer surface, respectively.]</nowiki>
```
Comment (happy5214) [inline standard_schema/prerelease/HED8.3.0.mediawiki:681]: Are `Perimeter` and/or `Circumference` appropriate as well? `Perimeter` might work well for an irregular or bordered shape.

Comment (happy5214) [reply standard_schema/prerelease/HED8.3.0.mediawiki:681]: (NB: I wanted to attach this to the `Size` line, but it was out of the diff's range.)

Comment (VisLab) [reply standard_schema/prerelease/HED8.3.0.mediawiki:683]: Added Perimeter

File: standard_schema/prerelease/HED8.3.0.mediawiki [hunk 3]
```diff
@@ -214,7 +214,7 @@ Each term in this vocabulary has a human-readable description and may include ad
 ***** Torso-back <nowiki>[The rear surface of the human body from the shoulders to the hips.]</nowiki>
 ***** Buttocks <nowiki>[The round fleshy parts that form the lower rear area of a human trunk.]</nowiki>
 ***** Torso-chest <nowiki>[The anterior side of the thorax from the neck to the abdomen.]</nowiki>
-***** Gentalia <nowiki>{deprecatedFrom=8.1.0}[The external organs of reproduction.]</nowiki>
+***** Gentalia <nowiki>{deprecatedFrom=8.1.0}[The external organs of reproduction. Deprecated due to spelling error. Use Torso/Genitalia.]</nowiki>
```
[the remark this answers is not in the document]
Comment (VisLab) [reply standard_schema/prerelease/HED8.3.0.mediawiki:217]: It was decided by the HWG that it shouldn't have been in there to start wtih.

File: README.md [hunk 4]
```diff
@@ -33,7 +33,7 @@ The following table summarizes the current versions of the HED schemas.
 | Schema | Latest<br/>version| Description | Prerelease<br/>version  | DOI  |
 | ------ | --------------- | ------- | ------------------------- | ---- |
 | [**standard**](./standard_schema) | 8.2.0 | Basic vocabulary for annotating data. | 8.3.0 | 10.5281/zenodo.7876037 |
-| [**score**](library_schemas/score) | 1.1.0  | SCORE standard vocabulary for <br/>clinical neurological annotation<br/>(See [**Score docs**](https://hed-schemas.readthedocs.io/en/latest/hed_score_schema.html.) |1.1.1 | 10.5281/zenodo.7897596  |
+| [**score**](library_schemas/score) | 1.1.0  | SCORE standard vocabulary for <br/>clinical neurological annotation<br/>(See [**Score docs**](https://hed-schemas.readthedocs.io/en/latest/hed_score_schema.html.).) |1.1.1 | 10.5281/zenodo.7897596  |
```
[the remark this answers is not in the document]
Comment (VisLab) [reply README.md:36]: Fixed in next commit

**Comment by VisLab** [comment]
As discussed with HWG -- this PR is being merged so that changes will be available in the prerelease schema viewer.  Comments are still possible as the schema itself is not being released yet.

**Comment by dorahermes** [comment]
I would like to add the different EEG artifact types to the main HED schema. Would you like me to open a pull request? 

Biological artifacts would include the following:

        ** Eye-blink-artifact <nowiki>[Example for EEG: Fp1/Fp2 become electropositive with eye closure because the cornea is positively charged causing a negative deflection in Fp1/Fp2. If the eye blink is unilateral, consider prosthetic eye. If it is in F8 rather than Fp2 then the electrodes are plugged in wrong. (Source: Beniczky ea 2013, Appendix S7; Beniczky ea 2017, Table 15.)]</nowiki>
        ** Eye-movement-horizontal-artifact <nowiki>[Example for EEG: There is an upward deflection in the Fp2-F8 derivation, when the eyes move to the right side. In this case F8 becomes more positive and therefore. When the eyes move to the left, F7 becomes more positive and there is an upward deflection in the Fp1-F7 derivation. (Source: Beniczky ea 2013, Appendix S7; Beniczky ea 2017, Table 15.)]</nowiki>
        ** Eye-movement-vertical-artifact <nowiki>[Example for EEG: The EEG shows positive potentials (50-100 micro V) with bi-frontal distribution, maximum at Fp1 and Fp2, when the eyeball rotated upward. The downward rotation of the eyeball was associated with the negative deflection. The time course of the deflections was similar to the time course of the eyeball movement. (Source: Beniczky ea 2013, Appendix S7; Beniczky ea 2017, Table 15.)]</nowiki>
        ** Slow-eye-movement-artifact <nowiki>[Slow, rolling eye-movements, seen during drowsiness. (Source: Beniczky ea 2013, Appendix S7.)]</nowiki>
        ** Nystagmus-artifact <nowiki>[Source: Beniczky ea 2017, Table 15.]</nowiki>
        ** Chewing-artifact <nowiki>[Source: Beniczky ea 2017, Table 15.]</nowiki>
        ** Sucking-artifact <nowiki>[Source: Beniczky ea 2017, Table 15.]</nowiki>
        ** Glossokinetic-artifact <nowiki>[The tongue functions as a dipole, with the tip negative with respect to the base. The artifact produced by the tongue has a broad potential field that drops from frontal to occipital areas, although it is less steep than that produced by eye movement artifacts. The amplitude of the potentials is greater inferiorly than in parasagittal regions; the frequency is variable but usually in the delta range. Chewing and sucking can produce similar artifacts. (Source: Beniczky ea 2013, Appendix S7; Beniczky ea 2017, Table 15.)]</nowiki>
        ** Rocking-patting-artifact <nowiki>[Quasi-rhythmical artifacts in recordings from infants caused by rocking/patting. (Source: Beniczky ea 2013, Appendix S7; Beniczky ea 2017, Table 15.)]</nowiki>
        ** Movement-artifact <nowiki>[Example for EEG: Large amplitude artifact, with irregular morphology (usually resembling a slow-wave or a wave with complex morphology) seen in one or several channels, due to movement. If the causing movement is repetitive, the artifact might resemble a rhythmic EEG activity. (Source: Beniczky ea 2013, Appendix S7; Beniczky ea 2017, Table 15.)]</nowiki>
        ** Respiration-artifact <nowiki>[Respiration can produce 2 kinds of artifacts. One type is in the form of slow and rhythmic activity, synchronous with the body movements of respiration and mechanically affecting the impedance of (usually) one electrode. The other type can be slow or sharp waves that occur synchronously with inhalation or exhalation and involve those electrodes on which the patient is lying. (Source: Beniczky ea 2013, Appendix S7; Beniczky ea 2017, Table 15.)]</nowiki>
        ** Pulse-artifact <nowiki>[Example for EEG: Occurs when an EEG electrode is placed over a pulsating vessel. The pulsation can cause slow waves that may simulate EEG activity. A direct relationship exists between ECG and the pulse waves (200-300 millisecond delay after ECG equals QRS complex). (Source: Beniczky ea 2013, Appendix S7; Beniczky ea 2017, Table 15.)]</nowiki>
        ** ECG-artifact <nowiki>[Example for EEG: Far-field potential generated in the heart. The voltage and apparent surface of the artifact vary from derivation to derivation and, consequently, from montage to montage. The artifact is observed best in referential montages using earlobe electrodes A1 and A2. ECG artifact is recognized easily by its rhythmicity/regularity and coincidence with the ECG tracing. (Source: Beniczky ea 2013, Appendix S7; Beniczky ea 2017, Table 15.)]</nowiki>
        ** Sweat-artifact <nowiki>[Is a low amplitude undulating waveform that is usually greater than 2 seconds and may appear to be an unstable baseline. (Source: Beniczky ea 2013, Appendix S7; Beniczky ea 2017, Table 15.)]</nowiki>
        ** EMG-artifact <nowiki>[Myogenic potentials are the most common artifacts. Frontalis and temporalis muscles (ex..: clenching of jaw muscles) are common causes. Generally, the potentials generated in the muscles are of shorter duration than those generated in the brain. The frequency components are usually beyond 30-50 Hz, and the bursts are arrhythmic. (Source: Beniczky ea 2013, Appendix S7; Beniczky ea 2017, Table 15.)]</nowiki>

Non-biological artifacts would include the following:

        ** Power-supply-artifact <nowiki>[50-60 Hz artifact. Monomorphic waveform due to 50 or 60 Hz A/C power supply. (Source: Beniczky ea 2013, Appendix S7; Beniczky ea 2017, Table 15.)]</nowiki>
        ** Induction-artifact <nowiki>[Artifacts (usually of high frequency) induced by nearby equipment (like in the intensive care unit). (Source: Beniczky ea 2013, Appendix S7; Beniczky ea 2017, Table 15.)]</nowiki>
        ** Dialysis-artifact <nowiki>[Source: Beniczky ea 2017, Table 15.]</nowiki>
        ** Artificial-ventilation-artifact <nowiki>[Source: Beniczky ea 2017, Table 15.]</nowiki>
        ** Electrode-pops-artifact <nowiki>[Brief discharges with a very steep upslope and shallow fall that occur in all leads which include that electrode. (Source: Beniczky ea 2013, Appendix S7; Beniczky ea 2017, Table 15.)]</nowiki>
        ** Salt-bridge-artifact <nowiki>[Typically occurs in 1 channel which may appear isoelectric. Only seen in bipolar montage. (Source: Beniczky ea 2013, Appendix S7; Beniczky ea 2017, Table 15.)]</nowiki>

**Pull request merged**